torch/onnx/symbolic_helper.py

```diff
@@ -226,17 +226,18 @@ def _sort_helper(g, input, dim, decending=True, out=None):
 def _topk_helper(g, input, k, dim, largest=True, sorted=False, out=None):
     if out is not None:
         _unimplemented("TopK", "Out parameter is not supported")
+    if not largest:
+        _unimplemented("TopK", "Ascending is not supported")
+    k = _maybe_get_const(k, 'i')
```

Comment:
Why move these 3 lines outside of the if branch? These are not required for the else branch

Reply:
Fixed